src/content/en/2019/fonts.md

```diff
@@ -11,120 +11,194 @@ last_updated: 2019-11-04T12:00:00+00:00:00
 
-- When pages uses Google Fonts, they use _a lot_ of Google fonts. They are free, after all. This seems like the most likely reason.
-- A lot of people are using Google Fonts with `@import` instead of `<link>`. My hunch is that this seems unlikely but maybe!
-- Google Fonts are being requested using an [HTTP `Link: ` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link) instead. Now we’re going off the deep end into very unlikely scenarios.
+### Are fonts being hosted on the same host or by a different host?
 
-**06.47: Uses a Google Fonts link first**
+Differentiating self hosting against third party hosting is increasingly relevant in an HTTP/2 world where the performance gap between a same-host and different-host connection can be wider. Same-host requests have the huge benefit of better potential for prioritization against other same-host requests in the waterfall.
 
-Google Fonts documentation encourages the `<link>` for the Google Fonts CSS to be placed as the first child in the `<head>` of a page. This is a big ask! In practice, this is not common as only half a percent of all pages took this advice. More so, if a page is using `preconnect` or `dns-prefetch` (read on for more about these) as `<link>` elements, these would come before the Google Fonts CSS anyway. In conclusion, 🤷‍♂️!
+Recommendations to mitigate the performance costs of loading web fonts from another host include `preconnect`, `dns-prefetch`, and `preload` but high priority web fonts (those used by very visually prominent content or body copy occupying the majority of a page) should be same-host requests to minimize the performance impact of web fonts.
 
-**06.01:  Are fonts being hosted on the same host or by a different host than the main content?**
+(Graph Title: Third Party Hosted versus Self hosted (local); Source: 06.01)
 
-This differentiation is important for performance reasons, increasingly relevant in an H2 world where the performance gap between a same-host and different-host connection can be wider. Same-host requests can also be better prioritized against same-host requests for other resources in the waterfall. Recommendations to mitigate this include `preconnect`, `dns-prefetch`, and `preload` but high priority web fonts (those used by a lot of content on your page) should be same-host requests to minimize the performance impact of web fonts.
+This is perhaps unsurprising given Google Fonts dominance in the next section (`06.02`), as Google Fonts takes the lion-share of 75% of web font requests, and all of those would be third-party requests. Note that `preload` is not yet available with Google Fonts.
```

Comment:
> Note that `preload` is not yet available with Google Fonts.
I missed something ?

Reply:
Yes I think this could do with more explanation @zachleat . Can you explain what you mean?

Reply:
Will put in this in as part of copyediting:

Note that `preload` is not yet available with Google Fonts, as they generate unique URLs for their fonts [which are liable to change](https://github.com/google/fonts/issues/1067), and even [serve different fonts for different browsers](https://github.com/google/fonts/issues/234).